src/main/java/cs2103/v15_1j/jimjim/command/SaveLocationCommand.java

```diff
@@ -20,10 +22,22 @@ public String getSavePath() {
     @Override
     public UIFeedback execute(ControllerStates conStates) {
     	String backup = conStates.config.savePath;
+    	
+    	try {
+    		File f = new File(backup);	
+    	} catch (Exception e) {
+    		return new FailureFeedback("Invalid file path specified!");
+    	}
+    	
+    	String extension = savePath.substring(savePath.length()-5, savePath.length()); 
+    	if (!extension.equals(".json")) {
```

Comment:
No need to check for the right extension. I don't see anything wrong if user wants to save it as a `txt` file. Also to check for extension you should use java's method instead of manually checking like this.